app/models/ability.rb

```diff
@@ -0,0 +1,18 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    return if user.nil?
+
+    @user = user
+    registered_user
+  end
+
+  def registered_user
+    can :manage, Expense
```

Comment:
given that this PR is about PeriodicExpenses (although this ability feature should have been in a different PR, actually), shouldn't there be a `:manage, PeriodicExpense` here as well?